src/components/global-logout/index.njk

```diff
@@ -0,0 +1,20 @@
+{% extends "common/layout/base-page.njk" %}
+{% set backLink = "SIGN_IN_HISTORY" | getPath %}
+{% block pageContent %}
+    <h1 class="govuk-heading-l govuk-!-margin-top-0 govuk-!-margin-bottom-3">{{ 'pages.globalLogout.title' | translate }}</h1>
+    <p class="govuk-body">{{ "pages.globalLogout.paragraph1" | translate }}</p>
+    <p class="govuk-body">{{ "pages.globalLogout.paragraph2" | translate }}</p>
+    <div class="govuk-warning-text">
```

Comment:
Could you use the [warning text](https://design-system.service.gov.uk/components/warning-text/) and [button](https://design-system.service.gov.uk/components/button/) components in this file please?